scripts/install.sh

```diff
@@ -3,15 +3,16 @@
 export SANDBOX_HOSTNAME=${SANDBOX_HOSTNAME:-openg2p.sandbox.net}
 export SPAR_HOSTNAME=${SPAR_HOSTNAME:-spar.$SANDBOX_HOSTNAME}
 
+COPY_UTIL=./copy_cm_func.sh
 NS=spar
+
 echo Create $NS namespace
 kubectl create ns $NS
 
-./copy_secrets.sh
+$COPY_UTIL secret postgres-postgresql postgres $NS
 
 kubectl -n $NS delete cm mapper-registry-schemas --ignore-not-found=true
 kubectl -n $NS create cm mapper-registry-schemas --from-file=schemas/FinancialAddressMapper.json
 
-helm -n $NS upgrade --install spar-self-service-ui openg2p/spar-self-service-ui -f values.yaml --set global.hostname=$SPAR_HOSTNAME $@
 
-helm -n $NS upgrade --install spar openg2p/social-payments-account-registry -f values.yaml --set global.hostname=$SPAR_HOSTNAME --wait $@
+helm -n $NS upgrade --install spar ../helm/charts/spar-mapper-api --set global.hostname=$SPAR_HOSTNAME --wait $@
```

Comment:
This is confusing.
This has to say either of the following:

- `--install spar ../helm/charts/spar -f values.yaml`
- `--install spar openg2p/spar -f values.yaml`

depending on what we want to achieve with this.
Because the purpose of this script is for installing spar as a whole, but the install.sh script only points to mapper here. Counterintuitive.
Do you want to have separate scripts to install mapper and self-service etc? I think that is definitely not required.
The self-service components are anyway disabled in values.yaml looks like, and whenever they are ready you can enable those back in values. So this looks like an unnecessary change.

Reply:
I was thinking of removing the spar folder from the charts. Like each component can be installed separately. The problem I see is that the values.yaml file is repeated like 3 times. I'm trying to reduce that into a simpler one.

What do you think?

Reply:
I would strongly advise against it. But if that's the plan you can go ahead with that, you will require multiple scripts to install the components.
But now that you mention it, I think spar/values.yaml should only contain:
```
global:
  hostname: spar.openg2p.sandbox.net

spar-self-service-api:
  enabled: true
  postgresql:
    enabled: false

# Choose between either of the following mappers. Or disable both if using external mapper.
spar-mapper-api:
  enabled: true
  fullnameOverride: mapper
  postgresql:
    enabled: false

spar-self-service-ui:
  enabled: true

sunbird-rc-g2p-mapper-registry:
  enabled: false
```